mountpoint-s3/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Unreleased
 
+### Other changes
+* Option `--profile <AWS_PROFILE>` no longer replaces the entire default credential chain.
```

Comment:
This is describing the implementation, not the customer-visible behavior. I'd just mention the issue we're fixing (credential process not working). Also, this is a breaking change -- we no longer validate incorrect profile names, and (if my other comment is right) also now respect env vars when profile is specified.

Reply:
You're absolutely right with the environment variables, that's why I ended up purging them from the test environment. Let me clarify that in the change log.

I'll also test the AWS CLI since I think you may be right about `--profile` being different from `AWS_PROFILE` in the CLI.

Reply:
CRT folks have added an option to disable ENV credentials on the default chain, so we can pick that up to avoid changing the behavior there. https://github.com/awslabs/aws-c-auth/commit/3d9f13fd6610d2c9f5ef5cf404bce4d9aeaca15b